src/app/components/fees-row/components/fees-row.layout.tsx

```diff
@@ -39,7 +39,7 @@ export function FeesRowLayout(props: FeesRowLayoutProps) {
         {feeField}
       </HStack>
       {isSponsored && <SponsoredLabel />}
-      {!meta.error && fieldWarning && <WarningLabel width="100%" >{fieldWarning}</WarningLabel>}
```

Comment:
I added this commit as I hadn't realised but [this pr](https://github.com/leather-wallet/extension/pull/4710) introduced a prettier problem